docs/manage/reference/viam-agent/manage-viam-agent.md

```diff
@@ -50,6 +53,60 @@ aliases:
 
   This command uninstalls `viam-agent`, `viam-server`, the machine cloud credentials file (<file>/etc/viam.json</file>), and the provisioning configuration file (<file>/etc/viam-provisioning.json</file>).
 
+{{% /tab %}}
+{{% tab name="Windows native" %}}
+
+On Windows, you can manage `viam-agent` using the Services GUI or the command line.
+
+## Manage `viam-agent` using the Services GUI
+
+1. Open the **Services** management console from your computer's start menu.
+
+1. Find `viam-agent` in the list of services.
+
+   {{<imgproc src="/manage/viam-agent-windows-services-manager.png" resize="x1100" declaredimensions=true alt="Windows Services manager with viam-agent highlighted." style="max-width:600px" class="shadow imgzoom" >}}
+
+1. Use the **Start Service**, **Stop Service**, and **Restart Service** buttons to manage `viam-agent`.
+
+1. To change the startup type of `viam-agent`, right-click on `viam-agent` and select **Properties**.
+   Select your desired startup type from the **Startup type** dropdown menu.
+
+   {{<imgproc src="/manage/startup-type-windows.png" resize="x1000" declaredimensions=true alt="Windows Services manager with viam-agent properties open." style="max-width:350px" class="shadow imgzoom" >}}
+
+## Manage `viam-agent` using the command line
```

Comment:
[q] why use sections within the Windows Native tab for CLI/GUI when those operations are (generally speaking) mutually exclusive? I would expect these parallel operations in tabs, but maybe we don't like stacking tabs that deep?

Reply:
It's a good question, and part of a larger convo about usage of tabs and expanders. I like both tabs and expanders but some people prefer seeing all the things in plain sight. Also our tabs aren't searchable with command f. But in this case it's all already in a tab and yes they are mutually exclusive so I will change to tabs!